xdsl/dialects/riscv.py

```diff
@@ -305,6 +303,28 @@ def verify(self) -> None:
         super().verify()
 
 
+@irdl_attr_definition
+class Shamt5Attr(IntegerAttr[IntegerType]):
```

Comment:
Please open a separate PR for this change

Reply:
Removed